en/_posts/2016-04-24-marathon-container-logs.md

```diff
@@ -10,6 +10,10 @@ It may be useful to send your application logs to the [PaaS Logs RunAbove lab](h
 
 #Inject my token into my container
 
-To be able to push logs into the PaaS Logs platform, you have to generate a new token from the RunAbove user interface. You will find more information on the [PaaS Logs quick start guide](https://community.runabove.com/kb/en/logs/quick-start.html). Once you have your token, you must inject it into your application containers using the label `USER_LOGS_TOKEN`, when creating/editing your app.
+To be able to push logs into the PaaS Logs platform, you have to generate a new token from the RunAbove user interface. You will find more information on the [PaaS Logs quick start guide](https://community.runabove.com/kb/en/logs/quick-start.html). Once you have your token, you must inject it into your application containers using the label `CONTAINERS_LOG_LOGS_TOKEN` (`USER_LOGS_TOKEN` is also supported), when creating/editing your app.
```

Comment:
PaaS Logs documentation have been migrated to OVH.com though redirections have been set up: https://docs.ovh.com/gb/en/mobile-hosting/logs-data-platform/quick-start/